.gitignore

```diff
@@ -1,8 +1,5 @@
 # Dependency directories
 node_modules/
 
-# Certora
-.certora*
-.last_confs
-certora_*
-resource_errors.json
+# Certora transients
+.certora/
```

Comment:
I think we should try to get all transient state into a single place

Reply:
checking this issue with python team